saml2/saml2-service-provider/src/test/java/org/springframework/security/saml2/provider/service/authentication/OpenSamlAuthenticationProviderTests.java

```diff
@@ -346,6 +350,28 @@ public void authenticateWhenDecryptionKeysAreWrongThenThrowAuthenticationExcepti
 		provider.authenticate(token);
 	}
 
+	@Test
+	public void checkThatAuthenticationIsSerializable() throws IOException {
```

Comment:
Please follow the test naming convention of `methodWhenConditionThenResult`. This could be something along the lines of `writeObjectWhenTypeIsSaml2AuthenticationThenNoException`.

Reply:
It's done.